### DISCUSSION TITLE
Could not acces either root  or magisk

### DISCUSSION THREAD

**Discussion by aioros57** [Fox2Code/FoxMagiskModuleManager#197] [General]
Hello every time there is a delay of fox manager once reboot . I have this message of the title in red magisk not install and zygisk similar.would be possible to see to fix this concern in the application, I am not the only one to have this problem see on XDA .

**Comment by androidacy-user** [comment]
I'm afraid I don't quite understand what you mean. Can you try rephrasing that?

**Comment by Fox2Code** [comment]
This should be posted in [issues](https://github.com/Fox2Code/FoxMagiskModuleManager/issues)
Also please give more information when reporting issues.
This is closed and locked as this is not posted in the right place.
